ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -718,13 +718,21 @@
           builders:
           - shell: |-
               #!/bin/bash
-              # gcloud auth login requires verification code from url
-              ${{GCLOUD_SDK_PATH}}/bin/gcloud auth activate-service-account --key-file=${{GCLOUD_KEY_PATH}}
+              set +x
```

Comment:
Better to add a comment here, explaining that set +x is for avoiding exposing credentials from console output.

Reply:
Sure, will update it.